examples/docker_provider/Gemfile

```diff
@@ -0,0 +1,5 @@
+source 'http://rubygems.org/'
+
+gem 'test-kitchen'
+gem 'kitchen-terraform'
```

Comment:
If we change this to source from the root of the project directory then we can drop dependency on `test-kitchen` and this will example can be used as additional integration tests.